plugins/kubernetes/app/views/kubernetes_release_groups/new.html.erb

```diff
@@ -37,7 +37,7 @@
               <%= build.project.name %> - <%= build.label %>
             </p>
           <% elsif @project.builds.any? %>
-            <%= form.select :build_id, @project.builds.order('id desc').limit(20).map { |b| [b.label, b.id] } %>
+            <%= form.select :build_id, @project.builds.order('id desc').limit(20).map { |b| [b.label, b.id] }, {}, { class: 'form-control' } %>
```

Comment:
The use of `@project` in here made me realise that the whole Kubernetes plugin is still not up-to-date with the changes introduced to the project level access control. We'll need to update the `before_action` filters in the controllers and remove the local `project` methods.